Tests/Marketplace/marketplace_services.py

```diff
@@ -754,7 +754,7 @@ def sign_pack(self, signature_string=None):
 
     @staticmethod
     def encrypt_pack(zip_pack_path, pack_name, encryption_key, extract_destination_path,
-                     private_artifacts_dir):
+                     private_artifacts_dir, new_encryption_key):
```

Comment:
Maybe refer to `new_encryption_key` as the `secondary_encryption_key`.

I've never been a fan of the whole first, second, third, nomenclature since inevitably we will have to change it again.